orientation/resources/16-tdd-first-steps.md

```diff
@@ -19,3 +19,19 @@ describe('add', () => {
 ```  
 
 It will fail, because _this is the first code you have written for you app_. There is no `add` method. Make that test pass by writing a function that does nothing, then rinse and repeat with every module until you have the skeleton of a project. Return to each method and write new tests for number of arguments, the type of data the function returns, whether the method returns anything at all, etc. Test multiple edge cases. What if no arguments are passed in? What if a string gets passed in where you need a number, or vice versa? What happens with negative numbers? Decimals? Leave no stone unturned!
+
+## Resources
+
+### package.json
+You will need to run `npm init -y` to create a package.json. Then run `npm install --save-dev mocha chai`
+
+Also, add the following to your package.json:
+```js
+  "scripts": {
+    "test": "mocha -R list --watch --recursive"
```

Comment:
May want to prompt Windows users to add the flag `--colors`. I didn't get colors on git bash until I did.